validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -913,12 +923,14 @@ def _inbound_challenge_authorization_callback(self, request, result,
             payload=payload,
             signature=signature,
             roles=[RoleType.Value("NETWORK")])
-
-        self.send(
-            validator_pb2.Message.AUTHORIZATION_CHALLENGE_SUBMIT,
-            auth_challenge_submit.SerializeToString(),
-            connection_id
-            )
+        try:
+            self.send(
+                validator_pb2.Message.AUTHORIZATION_CHALLENGE_SUBMIT,
+                auth_challenge_submit.SerializeToString(),
+                connection_id
+                )
+        except ValueError:
+            LOGGER.debug("Connection disconnected: %s", connection_id)
```

Comment:
This gets repeated a lot - can we put this in a `_safe_send` function?

Reply:
I will add this